_data-prepper/pipelines/configuration/sources/kinesis.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: kinesis
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# kinesis
+
+You can use `kinesis` source in Data Prepper to ingest records from one or more [Amazon Kinesis](https://aws.amazon.com/kinesis/) Data Streams. 
```

Comment:
Should we link here?

https://aws.amazon.com/kinesis/data-streams/

And then update the anchor to surround all of "Amazon Kinesis Data Streams?"

Reply:
Thanks @dlvenable. I have made the changes.